swap.conf

```diff
@@ -1,58 +1,36 @@
-# https://www.kernel.org/doc/Documentation/vm/zswap.txt
-
-zswap_enabled=1
-zswap_compressor=zstd     # lzo lz4 zstd lzo-rle lz4hc
-zswap_max_pool_percent=25 # 1-99
-zswap_zpool=z3fold        # zbud z3fold (note z3fold requires kernel 4.8+)
-
-################################################################################
-# ZRam
+# Entries in this file show the systemd-swap defaults as
+# specified in /usr/share/systemd-swap/swap.conf
+# You can change settings by editing this file.
+# Defaults can be restored by simply deleting this file.
 #
-# Kernel >= 3.15
-# Zram compression streams count for additional information see:
-# https://www.kernel.org/doc/Documentation/blockdev/zram.txt
+# See swap.conf(5) and /usr/share/systemd-swap/swap.conf for details.
+
+#zswap_enabled=1
+#zswap_compressor=zstd     # lzo lz4 zstd lzo-rle lz4hc
+#zswap_max_pool_percent=25 # 1-99
+#zswap_zpool=z3fold        # zbud z3fold (note z3fold requires kernel 4.8+)
 
-zram_enabled=0
-zram_size=$(( RAM_SIZE / 4 ))    # This is 1/4 of ram size by default.
-zram_count=${NCPU}               # Device count
-zram_streams=${NCPU}             # Compress streams
-zram_alg=zstd                    # See $zswap_compressor
-zram_prio=32767                  # 1 - 32767
+#zram_enabled=0
+#zram_size=$(( RAM_SIZE / 4 ))    # This is 1/4 of ram size by default.
+#zram_count=${NCPU}               # Device count
+#zram_streams=${NCPU}             # Compress streams
+#zram_alg=zstd                    # See $zswap_compressor
+#zram_prio=32767                  # 1 - 32767
 
-################################################################################
-# Swap File Chunked
-# Allocate swap files dynamically
-# For btrfs fallback to swapfile + loop will be used
-# ex. Min swap size 512M, Max 8*512M
-swapfc_enabled=0
-swapfc_force_use_loop=0     # Force usage of swapfile + loop
-swapfc_frequency=1          # How often to check free swap space
-swapfc_chunk_size=256M      # Allocate size of swap chunk
-swapfc_max_count=32         # Note: 32 is a kernel maximum
-swapfc_free_swap_perc=15    # Add new chunk if free < 15%
+#swapfc_enabled=0
+#swapfc_force_use_loop=0     # Force usage of swapfile + loop
+#swapfc_frequency=1          # How often to check free swap space
+#swapfc_chunk_size=256M      # Allocate size of swap chunk
+#swapfc_max_count=32         # Note: 32 is a kernel maximum
+#swapfc_free_swap_perc=15    # Add new chunk if free < 15%
                             # Remove chunk if free > 15+40% & chunk count > 2
-swapfc_path=/var/lib/systemd-swap/swapfc/
-# Only for swapfile + loop
-swapfc_nocow=1              # Disable CoW on swapfile
-swapfc_directio=1           # Use directio for loop dev
-swapfc_force_preallocated=0 # Will preallocate created files
+#swapfc_path=/var/lib/systemd-swap/swapfc/
+
+#swapfc_nocow=1              # Disable CoW on swapfile
+#swapfc_directio=1           # Use directio for loop dev
+#swapfc_force_preallocated=0 # Will preallocate created files
```

Comment:
This file should not have any comments in it (after the options that is) to match how systemd conf files are normally structured.